File: app/domain/entities/condiciontributaria.py
from typing import Optional, Any
from dataclasses import dataclass
from app.domain.entities.enums import AmbitoImpuestoEnum

@dataclass
class CondicionTributaria:
    id: Optional[int] = None
    nombre: Optional[str] = None
    descripcion: Optional[str] = None
    ambito: Optional[AmbitoImpuestoEnum] = None
    tipo_impuesto_id: Optional[int] = None
    tipo_impuesto: Optional[Any] = None

File: frontend/src/domain/entities/CondicionTributaria.ts
import type { AmbitoImpuestoEnum } from '../enums/AmbitoImpuestoEnum';

export interface CondicionTributaria {
    id: number;
    nombre: string;
    descripcion?: string;
    ambito?: AmbitoImpuestoEnum;
    tipo_impuesto_id?: number;
    tipo_impuesto?: any;
}
